app/frontend/tests/unit/views/About.spec.js

```diff
@@ -1,18 +1,35 @@
-import { shallowMount } from '@vue/test-utils';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
 import Vuetify from 'vuetify';
+import Vuex from 'vuex';
 
 import About from '@/views/About.vue';
 
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
 describe('About.vue', () => {
-  let vuetify;
+  let store;
+  const vuetify = new Vuetify();
 
   beforeEach(() => {
-    vuetify = new Vuetify();
+    store = new Vuex.Store();
+    store.registerModule('auth', {
+      namespaced: true,
+      getters: {
+        authenticated: () => true,
+        isAdmin: () => false,
+        keycloakReady: () => true,
+        // eslint-disable-next-line no-unused-vars
+        createLoginUrl: () => options => 'testurl'
```

Comment:
This can just be
```js
createLoginUrl: () => () => 'testurl'
```
Goal of this is to make it return a function which when called returns string 'testurl' - no need to specify a param when it's not used.